### ISSUE TITLE
Cannot get current scene

### ISSUE THREAD

**Issue by nateberkopec** [TeWu/krpc-rb#12]
What would the implementation for this look like?

Looking for [something like this](https://krpc.github.io/krpc/python/api/krpc.html#KRPC.current_game_scene).

**Comment by TeWu** [comment]
It is available in v0.4.0.beta2, both as hardcoded [`Core#current_game_scene`][1] method and as [dynamically generated][2] `KRPC#current_game_scene` method:

```ruby
#### Using KSP v1.2.2 and kRPC sever v0.4.0-127-g64fcf0c
[1] pry(main)> require 'krpc'
=> true
[2] pry(main)> KRPC::VERSION
=> "0.4.0.beta2"
[3] pry(main)> client = KRPC.connect
=> #<KRPC::Client:0x00000002d7de88 ...
[4] pry(main)> client.krpc.current_game_scene
=> :space_center
[5] pry(main)> client.core.current_game_scene
=> :space_center
```

It is not currently available in v0.3.x. I'll consider backporting it to v0.3.x soon.

[1]: https://github.com/TeWu/krpc-rb/blob/0.4.0.beta2/lib/krpc/service.rb#L97-L100
[2]: https://github.com/TeWu/krpc-rb/blob/0.4.0.beta2/lib/krpc/client.rb#L85-L124

**Comment by nateberkopec** [comment]
I think this is closed by 0.3.2, yes?

**Comment by TeWu** [comment]
Yes, v0.3.2 contain above mentioned backport, thus solving this issue for v0.3.x:

```ruby
#### Using KSP v1.3.0 and kRPC server v0.3.9
[1] pry(main)> require 'krpc'
=> true
[2] pry(main)> KRPC::VERSION
=> "0.3.2"
[3] pry(main)> client = KRPC.connect
=> #<KRPC::Client:0x000000026952d8 ...
[4] pry(main)> client.krpc.current_game_scene
=> :space_center
[5] pry(main)> client.core.current_game_scene
=> :space_center
```
Thanks for reporting this issue!
